checks/raw/fuzzing.go

```diff
@@ -15,14 +15,48 @@
 package raw
 
 import (
+	"bytes"
 	"fmt"
+	"regexp"
+	"strings"
 
 	"github.com/ossf/scorecard/v4/checker"
 	"github.com/ossf/scorecard/v4/checks/fileparser"
 	"github.com/ossf/scorecard/v4/clients"
 	sce "github.com/ossf/scorecard/v4/errors"
 )
 
+const (
+	fuzzNameOSSFuzz         = "OSSFuzz"
```

Comment:
this used to be an enum. I think you can change `fuzzNameOSSFuzz` -> `fuzzerOSSFuzz`, `fuzzNameClusterFuzzLite` -> `fuzzerClusterFuzzLite`, etc

Reply:
SGTM, resolved